src/com/projectkorra/projectkorra/util/DamageHandler.java

```diff
@@ -65,12 +70,17 @@ public static void damageEntity(final Entity entity, Player source, double damag
 				((LivingEntity) entity).damage(damage, source);
 				final double nextHealth = ((LivingEntity) entity).getHealth();
 				entity.setLastDamageCause(finalEvent);
-				if (ignoreArmor) {
-					if (finalEvent.isApplicable(DamageModifier.ARMOR)) {
-						finalEvent.setDamage(DamageModifier.ARMOR, 0);
+				
+				if (ignoreArmor && hasArmor((LivingEntity) entity)) {
+					if(CooldownDamage().get((LivingEntity) entity)==null) {
+						((LivingEntity) entity).setHealth(prevHealth-damage);
```

Comment:
Setting an entity's health evades the damage event, which will cause issues for both ProjectKorra and other plugins. You should be setting the damage to be dealt within `finalEvent`